src/Workspaces/CSharpTest/Formatting/FormattingTests.cs

```diff
@@ -9603,5 +9603,43 @@ void M([My][My] int x)
 
             await AssertFormatAsync(expectedCode, code);
         }
+
+        [Fact, WorkItem(47442, "https://github.com/dotnet/roslyn/issues/47442")]
+        [Trait(Traits.Feature, Traits.Features.Formatting)]
+        public async Task IndentImplicitObjectCreationInitializer()
+        {
+            var code = @"
+class C
+{
+    public string Name { get; set; }
+    public static C Create1(string name)
+        => new C()
+    {
+        Name = name
+    };
+    public static C Create2(string name)
+        => new()
+    {
+        Name = name
+    };
+}";
```

Comment:
How are these two cases different?  (Asking as they look identical to be)

Reply:
just to make sure we match `new C()` formatting, however that may be.

Reply:
Ah got it. Thanks. I totally didn't see that :-)